PUBLICATION.md

```diff
@@ -0,0 +1,133 @@
+
+The purpose of this guide is to ensure consistency, professionalism, and proper
+attribution in all citations and references to our published reports and
+findings. These guidelines are intended to support both internal and external
+stakeholders in referencing and utilizing our work responsibly.
+
+This guide applies to all materials published by Informal Systems, Inc.,
+including but not limited to:
+
+* Security audit reports
+* Technical whitepapers 
+* Blog posts and technical write-ups 
+* Conference presentations and materials
+* Posts on Twitter, LinkedIn, Farcaster, or other social media sites
+
+By following these guidelines, we aim to:
+
+* Establish a clear and professional citation standard for our work. 
+* Facilitate proper recognition of our contributions to the blockchain security community. 
+* Prevent misuse or misrepresentation of our findings.
+
+These guidelines are not exhaustive but provide a foundation for citing and
+publishing our materials. For specific questions or additional permissions,
+please contact your project manager.
+
+## Citation Format
+
+To ensure proper attribution and consistency when referencing materials
+published by Informal Systems, Inc., please use the following citation format:
+
+**For Security Audit Reports:**
+
+```
+[Author(s)], "[Report Title]," Informal Systems, Inc., [Month Year], available at: [URL]
+```
+
+*Example:*
+
+```
+Jane Doe, "Security Audit Report for ABC Protocol," Informal Systems, Inc., December 2024, available at: https://github.com/informalsystems/audits/blob/main/abc-protocol/2024-06-01%20Audit%20Report%20-%20ABC%20Protocol.pdf
+```
+
+**For Technical Whitepapers:**
+
+```
+[Author(s)], "[Paper Title]," Informal Systems, Inc., [Month Year], available at: [URL]
+```
+
+*Example:*
+
+```
+John Smith, "A Deep Dive into IBC Security," Informal Systems, Inc., January 2024, available at: https://informal.systems/whitepapers/ibc-security
+```
+
+**For Blog Posts:**
+
+```
+[Author(s)], "[Blog Title]," Informal Systems, Inc., [Month Year], available at: [URL]
+```
+
+*Example:*
+
+```
+Alex Johnson, "Five Tips for Writing Secure Cosmos SDK Code," Informal Systems, Inc., November 2024, available at: https://informal.systems/blog/cosmos-sdk-security-tips
+```
+
+Ensure that the citation includes all relevant authors and links to the most
+recent and official version of the publication. If uncertain, please consult
+your project manager.
+
+## Attribution Guidelines
+
+When citing or referencing materials published by Informal Systems, Inc., it is
+critical to ensure accurate and appropriate attribution. Below are the
+guidelines for proper attribution:
+
+- **Do Not Generalize Findings:** Our security audit reports are limited to the specific scope and components reviewed at the time of the audit. They should not be used to assert or imply that an entire codebase or system is secure. Instead, references should explicitly note the scope of the audit as defined in the report.
```

Comment:
It should be mentioned in the "Do Not Generalize Findings" paragraph (or in a separate one) that our assessments are limited to the specific review methodology. For instance, one cannot say " Informal has formally verified..." when we just use the Quint's simulator.

Reply:
Also, perhaps it makes sense to make clear here that scope also implies a specific version of the software under review. We say here "at the time of the audit", while the "time" is less relevant than the specific artifact and software version we considered. In particular if the client continues development in parallel to our audit (and thus effectively writes new version of the software), this might be confusing to some people.